Kitodo/src/main/java/org/kitodo/production/helper/SearchResultGeneration.java

```diff
@@ -70,7 +70,7 @@ public HSSFWorkbook getResult() {
         List<ProcessDTO> processDTOS = new ArrayList<>();
         try {
             processDTOS = ServiceManager.getProcessService().findByQuery(getQueryForFilter(ObjectType.PROCESS),
-                ServiceManager.getProcessService().sortByTitle(SortOrder.ASC), false);
+                    ServiceManager.getProcessService().sortById(SortOrder.ASC),false);
```

Comment:
`false` needs a white space in front and maybe this must be changed to `true` as we / I got time out issues with ` false`

Reply:
right, but this is a different issue ;-)
This PR is only about ordering.
I will add the whitespace